Tahi-project/tahi
# Remove deploy:cold target

When starting with a blank database, automatically run a schema load rather than
a migration.

It's really annoying when testing things via vagrant to have to remember to run `cap vagrant deploy:cold`. This fix avoids that.

To test:

1. Create a fresh vagrant VM for aperta
2. Run `cap vagrant deploy` in this branch.
3. Ensure that the schema is loaded, not migrated from the beginning of time.

---

#### Code Review Tasks:

Reviewer tasks:


#### After the Code Review:

Author tasks:
